scripts/push-to-assets-branch

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+rm -rf /tmp/tmp-assets
+mkdir -p /tmp/tmp-assets
+cp -r assets/ index.yaml index.html sha256sum/ /tmp/tmp-assets
```

Comment:
copy what's has generated from dev-v2.5 branch and paste into tmp dir and eventually asset branch. Then commit and push.